lib/saluki-components/src/sources/dogstatsd/mod.rs

```diff
@@ -52,6 +52,9 @@ use self::framer::{get_framer, DsdFramer};
 mod origin;
 use self::origin::{origin_info_from_metric_packet, OriginEnrichmentConfiguration};
 
+mod filters;
+use filters::{EnablePayloadsFilter, Filter};
```

Comment:
Small nit: I/we prefer to qualify child modules via `self` just to avoid _any_ confusion about them being an external crate or not.

```suggestion
use self::filters::{EnablePayloadsFilter, Filter};
```